index.html

```diff
@@ -3484,11 +3488,11 @@ <h2>
             </dt>
             <dd>
 If the dereferencing is successful, this MUST be a <a href="metadata-structure">
-metadata structure</a>. This structure contains
-metadata about the <code>content</code>.  
-Additionally, in the case that the <a>DID URL</a> is a <a>DID</a>, then this
-MUST be a <code>did-document-metadata</code> as described in <a>DID Resolution</a>,
-obtained through the resolution process.
+metadata structure</a>, but the structure MAY be empty. This structure contains
+metadata about the <code>content-stream</code>.  
+Especially in the case that the <a>DID URL</a> is a <a>DID</a>, this MAY be a
```

Comment:
```suggestion
If the <code>content-stream</code> is a <a>DID document</a>, this MUST be a
```

I think this is a more reliable way of describing the same idea..

Reply:
Thanks, agreed!